permission/src/main/java/org/wildfly/security/permission/LongNameSetPermissionCollection.java

```diff
@@ -94,6 +90,10 @@ public boolean hasMoreElements() {
             return bits != 0;
         }
 
+        private Permission permissionFor(int id) {
```

Comment:
@nidhiazad Just a total minor, for code style, can you please move this `private` method under all the public methods? Otherwise LGTM!

Reply:
sure. Changes done